clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -938,6 +939,8 @@ private void handleResponses(long now, List<ClientResponse> responses, Map<Strin
                 if ((calls == null) || (!calls.remove(call))) {
                     log.error("Internal server error on {}: ignoring call {} in correlationIdToCall " +
                         "that did not exist in callsInFlight", response.destination(), call);
+
+                    System.exit(1);
```

Comment:
Hmm, looks like you were using this during debugging and forgot to remove it?

Reply:
yup, thanks for the catch.